packages/bpk-tokens/src/web/base/aliases.json

```diff
@@ -33,7 +33,7 @@
     "ZINDEX_AUTOSUGGEST": "900",
     "ZINDEX_POPOVER": "900",
     "ZINDEX_TOOLTIP": "900",
-    "ZINDEX_MODAL_SCRIM": "1000",
+    "ZINDEX_SCRIM": "1000",
```

Comment:
Unfortunately I dont think it's worth renaming tokens as part of this change - tokens are a public API in `bpk-tokens` & `bpk-mixins` and we cannot be sure of who is using what. Ok to keep them as is? Either that or we deprecate the existing names and duplicate them with new names.

Reply:
Right, I see

I'm happy to go down the duplicate and deprecate route, unless you think it's too much of a pain?

Reply:
Nah I think thats OK - we haven't done it before, but we need some sort of policy for deprecating older tokens. There is a meta property on tokens that we could assign a `deprecated: true,` property to.

Reply:
I've updated the files, so most variables are back. This alias though gets output directly in some of the files (eg. packages/bpk-tokens/tokens/travelpro.raw.json) and if I add it back I don't think it can be properly marked as deprecated. Should I leave it as is?

Reply:
Hmm good point - let me get back to you...

Reply:
We've agreed that it's not part of the public interface, so this is fine!

Reply:
cool